site/docs/data.md

```diff
@@ -45,7 +43,7 @@ Here is a list of all properties describing a `data` source from URL:
 
 {% include table.html props="url,format" source="UrlData" %}
 
-For example, the following specification loads data from a relative `url`: `data/cars.json`. Note that the format type is implicitly json by default.
+For example, the following specification loads data from a relative `url`: `data/cars.json`. Note that the format type is implicitly `json` by default.
```

Comment:
> Note that the format type is implicitly `json` by default.

Didn't we read extension name or something? 

In any case we use `"json"` for string values and `key` for keys.